### DISCUSSION TITLE
How to convert a 1080 full hd image with text in center of black background

### DISCUSSION THREAD

**Discussion by eddievalentin** [ImageMagick/ImageMagick#6773] [Help]
### ImageMagick YouTube

ImageMagick 7 

### Operating system, version and so on

Windows 11

### Description

How to convert a 1080 full hd image with text in center of black background. What parameter/command line to use especially for YouTube?

**Comment by fmw42** [comment]
What result do you want?  What is wrong with the image?

**Reply by eddievalentin** [reply]
Im currently using this parameter:

C:\ImageMagick\convert.exe -background black -fill white -font Courier -pointsize 72 -gravity center label:"Text Here" -extent 1920x1080 image2.png

the thing is everytime i upload still image video with audio to YouTube doesn't sound right/lossless. Can You help? and also with bit depth.

**Comment by snibgo** [comment]
Your question seems to be about YouTube. I suggest you read their documentation, such as https://support.google.com/youtube/troubleshooter/2888402
